css/style.css

```diff
@@ -0,0 +1,70 @@
+*{
+    margin:0
+  
+}
+body{
+    background-color:rgb(179, 180, 187);
+}
+
+nav {
+    margin: auto;
+    display: block;
+    position: fixed;
+
+}
+.menu {
```

Comment:
Great use of the html tag within css!!!
Remember not to leave any blank lines before the closing curly bracket.

Reply:
- [ ] You did a good job working on your reviews! 💯 However, in order to improve this one please consider adding an explanation. Think about "How this change could improve the code?". 🤓 🔧